src/Drivers/EodApi.php

```diff
@@ -68,19 +68,29 @@ public function getBatch(array $symbols)
     {
         $symbol  = array_shift($symbols);
```

Comment:
En die eerste symbol mag niet in het `s` veld zitten bij de request?

Wat een brakke api :p

Reply:
Die eerste symbol zit al in de url verwerkt, je stuurt 1 symbol mee in een request naar EOD, en de rest van de symbols waar je in 1 keer data van wilt geef je mee in het s veld, dus bij 1 symbol is het niet nodig om het s veld te gebruiken.